src/GlobalScreen/Scope/Layout/Collector/MainLayoutCollector.php

```diff
@@ -133,6 +141,19 @@ public function getFinalPage() : Page
             // PAGE
             $page_modification = $provider->getPageBuilderDecorator($called_contexts);
             $this->replaceModification($final_page_modification, $page_modification, PageBuilderModification::class);
+
+            if(get_class($provider) !== "ILIAS\GlobalScreen\Provider\GSModificationProvider") { //TODO: raus
```

Comment:
@chfsx , do you have a good idea how to get rid of of this?
Id rather not extend interfaces/base classes...

Reply:
I still don't quite understand why this if query is needed here. Will you enlighten me please? :-)

Reply:
I now extended ModificationProvider-interface; works like a charme ;)